src/js/components/Avatar/__tests__/Avatar-test.tsx

```diff
@@ -87,7 +87,9 @@ describe('Avatar', () => {
   test('icon renders', () => {
     const { container } = render(
       <Grommet>
-        <Avatar src={<Favorite color="accent-2" />} background="accent-4" />
```

Comment:
We need both, a test with an element `src` as well as children. Taking a look at propTypes.js, it looks like index.d.ts is missing a type for `src`.

Reply:
In the documentation on grommet-site `src`'s only type is a string and this is reflected in the index.d.ts. However, the propTypes file has `src: PropTypes.oneOfType([PropTypes.string, PropTypes.element])` so I think it is the propTypes file that is incorrect.

I tested `<Avatar src={<Favorite color="accent-2" />} background="accent-4" />` and it wasn't showing up correctly. Whereas the code below was showing up correctly. [Example here](https://codesandbox.io/s/grommet-v2-template-forked-z91x2?file=/index.js).
```
<Avatar background="accent-4">
  <Favorite color="accent-2" />
</Avatar>
```

Reply:
OK. I had just noticed the discrepancy. Thanks for delving deeper.

Reply:
Still need to clean up `propTypes.js`

Reply:
To add to this, the old snapshot never contained an SVG for the icon, where now it does. I'll remove `PropTypes.element` from the `propTypes` file.